string[] boxes_id
---
bool success
string message